textdescriptives/components/quality.py

```diff
@@ -469,12 +469,12 @@ def set_extensions(self):
         """Set required extensions."""
 
         for ext_name, span_getter in self.extensions.items():
-            doc_getter = span_getter_to_doc_getter(span_getter)
+            # doc_getter = span_getter_to_doc_getter(span_getter)
 
             if not Span.has_extension(ext_name) or self.force is True:
                 Span.set_extension(ext_name, getter=span_getter, force=True)
             if not Doc.has_extension(ext_name) or self.force is True:
-                Doc.set_extension(ext_name, getter=doc_getter, force=True)
+                Doc.set_extension(ext_name, getter=span_getter, force=True)
```

Comment:
we should add a test which check that the whole doc span and doc metrics are consistent `doc[:]._.metric == doc._.metric`.

Reply:
We can actually do this as a general check for all pipelines